### DISCUSSION TITLE
Carousel last item snaps at wrong position

### DISCUSSION THREAD

**Discussion by AxelReid** [mantinedev/mantine#3135] [Q&A]
The carousel in this [site](https://car-rent-with-mantine.vercel.app/) as an example, when reaching the last slide it doesn't stop at an expected posotion. If in the Carousel viewport there're 3 slides, there'll be 2 slides gap at the end but i think shouldn't be and should stop  sliding after last item enters...

**Comment by rtivital** [comment]
You should check embla docs/issues for this. It is not controlled on our side.
